home/models.py

```diff
@@ -1004,12 +1014,16 @@ def tag(self):
         return list(self.tags.all())
 
     @short_description("Whatsapp Body")
-    def wa_body(self):
+    def wa_body(self) -> str:
         body = (
-            "\n".join(m.value["message"] for m in self.whatsapp_body)
+            "\n".join(
+                m.value["message"] if is_subscriptable(m.value) else ""
+                for m in self.whatsapp_body
+            )
             if self.whatsapp_body
             else ""
```

Comment:
If it's a template, do we instead want to show the template text here instead? I think this is used in list views of content pages.